src/containers/picklist/PickListContainer.js

```diff
@@ -66,9 +66,9 @@ export default class PickListContainer extends React.Component {
   moveModal = (toId, toName) => {
     const moveFunction = this.isTypeNode() ? this.props.moveNode : this.props.moveObject
     const callback = this.isTypeNode() ? this.nodeCallback(toName) : this.objectCallback(toName)
-    this.state.itemsToMove.forEach((itemToMove => {
-      moveFunction(itemToMove.id, toId, this.props.user.id, callback)
-    }))
+    const arr = [];
```

Comment:
Rule #1 in functional programming, avoid mutability. Therefore do this instead. And save yourself a line:

const arr = this.state.itemsToMove.map(itemToMove => itemToMove.id)

also rename the arr variable to something more readable like toMove